### ISSUE TITLE
Configurable line wrapping in code blocks

### ISSUE THREAD

**Issue by fabiospampinato** [notable/notable#985]

**Comment by RyanGreenup** [comment]
Yep, this would be great, not all languages make it pleasant to do things over multiple lines, e.g. `bash` can be a pain when you specify many arguments to a function that are over multiple lines, because commenting after the `\` will interrupt it.

This is what it looks like when you use the [`minted`](https://github.com/gpoore/minted) package:

![image](https://user-images.githubusercontent.com/47832594/72765167-11c59d80-3be3-11ea-94ab-dd24ed2e877b.png)

Although this may be handled just as well by implementing a button to pass the MD to pandoc with the option to specify a $\LaTeX$ header and then inserting the note as attachment, because usually this is something that needs to be done before printing anyway.
